certified-connectors/Repfabric Job Loader/apiDefinition.swagger.json

```diff
@@ -0,0 +1,207 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Repfabric Job Loader",
+    "description": "This Connector allows you to create data such as companies, contacts, opportunities, and more in Repfabric right from Excel. Rather than tediously uploading formatted data on spreadsheets, select the rows of data you want to appear in Repfabric, run your flow, and instantly your information will appear!",
+    "contact": {
+      "name": "Repfabric Support",
+      "url": "https://repfabric.com/",
+      "email": "support@repfabric.com"
+    },
+    "version": "1.0"
+  },
+  "host": "yz1yygr342.execute-api.us-east-2.amazonaws.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": ["application/json"],
+  "produces": [],
+  "paths": {
+    "/default/jobs": {
```

Comment:
Hello @andreroberts1119, 

it is not allowed to have a default response only. please define a proper response. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).

Reply:
@vmanoharas This has been adjusted.